src/instructlab/schema/taxonomy.py

```diff
@@ -429,6 +429,6 @@ def parse(self, path: str | pathlib.Path) -> Taxonomy:
 
             self._schema_validate(text=text, taxonomy=taxonomy)
         except Exception as e:
-            raise TaxonomyReadingException from e
+            raise TaxonomyReadingException(f"{e}") from e
```

Comment:
Isn't this more simply:

```suggestion
            raise TaxonomyReadingException(str(e)) from e
```